### PR TITLE
Added 202 response code for patch async operations

### PR DESCRIPTION
**Pull request by akashkeshari**
The PR is to avoid issues caused by the breaking change that rpaas is going to introduce
https://teams.microsoft.com/l/message/19:05524c1405d64237856c419bf41fd2f7@thread.skype/1620325548526?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=929a4be2-b219-4f5f-8dd0-94ca0eeec6b5&parentMessageId=1620325548526&teamName=RP-as-a-Service%20Partners&channelName=Announcements&createdTime=1620325548526

Related icm link: https://portal.microsofticm.com/imp/v3/incidents/details/239962688/home

### CODE REVIEW

File: specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-04-01-preview/connectedClusters.json [hunk 1]
```diff
@@ -149,6 +149,12 @@
               "$ref": "#/definitions/ConnectedCluster"
             }
           },
+          "202": {
+            "description": "Resource patch request accepted",
```
Comment (rapatchi) [inline specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-04-01-preview/connectedClusters.json:153]: do we need to add this in other api versions as well ?.

Comment (akashkeshari) [reply specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-04-01-preview/connectedClusters.json:153]: No, we only need to add in the api-versions where patch async operations is supported.

**Review by rapatchi — APPROVED** [review]

**Review by JeffreyRichter — APPROVED** [review]

File: specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-04-01-preview/connectedClusters.json [hunk 2]
```diff
@@ -149,6 +149,12 @@
               "$ref": "#/definitions/ConnectedCluster"
             }
           },
+          "202": {
```
Comment (mentat9) [inline specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-04-01-preview/connectedClusters.json:152]: >202 [](http://example.com/codeflow?start=11&length=3)

This requires x-ms-long-running-operation extension.

Comment (akashkeshari) [reply specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-04-01-preview/connectedClusters.json:146]: Added

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Pull request merged**